### PR TITLE
Voting portal post-launch small fixes

### PR DESCRIPTION
**Pull request by Will-Howard**
See latest [designs](https://www.figma.com/file/OWruitRTyWpmLJjaT4RedA/Voting-portal?node-id=455%3A5055&mode=dev)

In this PR:
- Make the header less confusing by having "Voting portal" as a subtitle rather than the title (note I've reused the regular subtitle component, which differs from the designs but I think makes more sense for consistency)
- Use arrows instead of bullets in the thank you page list
- Change the styling of the button to set the flair on your profile after you click it
- Making the warning for logged out and ineligible users clearer

Screenshot 2023-12-04 at 16 13 10
Screenshot 2023-12-04 at 16 33 08
For this warning, I'm not really trying to make it look great as we don't link to the voting portal for logged out users. I'm just trying to make it very obvious as soon as they visit the page:
Screenshot 2023-12-04 at 16 54 16
Screenshot 2023-12-04 at 16 54 53

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1206095642569357) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
Looks really great! 😍 

made a small tweak so the text fits on one line in the button (for "Remove icon from you profile")
Screenshot 2023-12-04 at 18 30 19

File: packages/lesswrong/components/ea-forum/voting-portal/VotingPortalThankYou.tsx [hunk 1]
```diff
@@ -214,34 +228,37 @@ const VotingPortalThankYou = ({currentUser, classes}: {
...
               : "Add icon to your profile"}</>
           }
         </div>
       </div>
       <div className={classes.hr} />
-      <ul className={classes.list}>
-        <li>
+      <div className={classes.list}>
+        <div className={classes.listRow}>
+          -&gt;
```
Comment (jpaddison3) [inline packages/lesswrong/components/ea-forum/voting-portal/VotingPortalThankYou.tsx:245]: Uhhhh. We have a bleeping ligature for this? I mean, if we have it, I guess use it, but seems kinda insane.

**Review by jpaddison3 — APPROVED** [review]

File: packages/lesswrong/components/ea-forum/voting-portal/VotingPortalThankYou.tsx (hunk 1 above)
Comment (Will-Howard) [reply packages/lesswrong/components/ea-forum/voting-portal/VotingPortalThankYou.tsx:245]: Yeah Ollie introduced me to this, and it is kind of insane. It only seems to work in some elements (e.g. it doesn't work in buttons), I'm on the fence about using it in general but in this case the "ArrowRight" icon looked too narrow whereas this is just right so I decided to go with it

File: packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx [hunk 2]
```diff
@@ -322,8 +322,9 @@ const GivingSeasonHeader = ({
                             {lightbulbIcon}
                           </div>
                         )}
-                        {isVotingPortal ? "Voting portal" : forumHeaderTitleSetting.get()}
+                        {isVotingPortal ? forumShortTitleSetting.get() : forumHeaderTitleSetting.get()}
                       </Link>
+                      <span className={classes.hideMdDown}><HeaderSubtitle /></span>
```
[the remark this answers is not in the document]
Comment (Will-Howard) [reply packages/lesswrong/components/ea-forum/giving-portal/GivingSeasonHeader.tsx:327]: I'm going to leave it as is, no strong reason

**Pull request merged**